qabelbox/src/main/java/de/qabel/qabelbox/services/QabelService.java

```diff
@@ -298,4 +164,23 @@ private void setNotification(String text) {
 
         mNotificationManager.notify(SERVICE_NOTIFICATION_ID, mBuilder.build());
     }
+
+    // TODO: Needs to be called from LocalQabelService via EventBus
```

Comment:
We might also use the same concept as in the IncomingHandlerThread which delivers DropMessages to other applications. But that's not necessary.